### PR TITLE
Makefile:  Support arbitrary Avocado location

### PR DESCRIPTION
**Pull request by rdkdd**
If Avocado is installed in non-standart way (e.g. in virtual
environment), `make link` command fails for two reasons:
1. It does not create symlinks of avocado-vt config files.
2. Sometimes (e.g. while in virtual evironment) setuptools `--user`
is not available.

One could simply define AVOCADO_DIRNAME env variable before `make link`.

### CODE REVIEW

**Comment by rdkdd** [comment]
@ldoktor @clebergnu could you please check and possibly merge?
thank you!

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,5 @@
 PYTHON=$(shell which python)
+PYTHON_DEVELOP_ARGS=$(shell if ($(PYTHON) setup.py develop --help 2>/dev/null | grep -q '\-\-user'); then echo "--user"; else echo ""; fi)
```
Comment (ldoktor) [inline Makefile]: the `PYTHON_DEVELOP_ARGS` is not really related to this commit. Would you please extract it and add in a separate commit?

**Comment by rdkdd** [comment]
thanks
